sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client_helpers.py

```diff
@@ -0,0 +1,1217 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from io import BytesIO
```

Comment:
You shouldn't need this file in this review anymore? Might be helpful to delete it in this branch to avoid merge conflicts later.

Reply:
container_client_helpers.py uses `_generic_delete_blob_options` which lives in blob_client_helpers.py